File: Php/app/Chart/ChartColumnModel.php
namespace app\Chart{

	class ChartColumnModel {

		public $type;
		public $label;
		public $pattern;

		public function __construct($type, $label, $pattern = null){
			$this->type = $type;
			$this->label = $label;
			$this->pattern = $pattern;
		}

	}

}

File: Web/Models/ChartModels/ChartColumnModel.cs
using JetBrains.Annotations;
using Newtonsoft.Json;

namespace Web.Models.ChartModels
{
	public class ChartColumnModel
    {
        [UsedImplicitly]
        [JsonProperty("type")]
		public string Type { get; private set; }

        [UsedImplicitly]
        [JsonProperty("label")]
        public string Label { get; private set; }

        [UsedImplicitly]
        [JsonProperty("pattern")]
	    public string Pattern { get; private set; }

	    public ChartColumnModel(string type, string label, string pattern = null)
        {
			Type = type;
			Label = label;
			Pattern = pattern;
		}
	}
}